test/cpp_extensions/open_registration_extension/torch_openreg/tests/test_device.py

```diff
@@ -27,6 +27,13 @@ def test_device_context(self):
             self.assertEqual(torch.accelerator.current_device_index(), 1)
         self.assertEqual(torch.accelerator.current_device_index(), device)
 
+    def test_device_capability(self):
+        capability = torch.accelerator.get_device_capability("openreg:0")
+        supported_dtypes = capability["supported_dtypes"]
+        expected_dtypes = [torch.float32, torch.float64, torch.int32, torch.int64]
```

Comment:
The dtype in `expected_dtypes` should be exactly the same as the dtype in `supported_dtypes`.